### PR TITLE
Home view has been updated following the current design of the web

### PR DESCRIPTION
**Pull request by Samuel-AQ**
## Proposed changes

Home view has been updated following the current design of the web

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

There is much more to do, this is just the view. The logic and the refactor will be applied soon.

### CODE REVIEW

**Comment by javidiazdom** [comment]
Great job! At a first glance the new page on storybook looks really nice. Ill take a look now at the code.

File: frontend/src/infrastructure/view/components/atoms/Image/Image.tsx [hunk 1]
```diff
@@ -2,8 +2,11 @@ import * as React from 'react';
...
 
-export const Image: React.FC<ImageProps> = ({source, description= 'image', width}) => (
-  <img src={source} alt={description} width={width}/>
-);
+export const Image: React.FC<ImageProps> = ({
+  source,
+  description = 'image',
+  width,
+  cssClass,
+}) => <img src={source} alt={description} width={width} className={cssClass} />;
```
Comment (javidiazdom) [inline frontend/src/infrastructure/view/components/atoms/Image/Image.tsx]: Why pass the CSS class and height as parameters? This is not needed. Styles can be applied on parent components where this component is used.

File: frontend/src/infrastructure/view/components/atoms/Image/types.ts [hunk 2]
```diff
@@ -2,4 +2,6 @@ export interface ImageProps {
   source: string;
   description: string;
   width?: string;
+  height?: string;
+  cssClass?: string;
```
Comment (javidiazdom) [inline frontend/src/infrastructure/view/components/atoms/Image/types.ts]: Same as before. Unnecesary parameters?

**Review by javidiazdom — CHANGES_REQUESTED** [review]
The homepage design is awesome. Just take a look at that parameters on the Image component and correct me if Im wrong, of course.

File: frontend/src/infrastructure/view/components/atoms/Image/Image.tsx (hunk 1 above)
Comment (Samuel-AQ) [reply frontend/src/infrastructure/view/components/atoms/Image/Image.tsx]: Sorry, I wrote that in order to test something I wanted to test but in the end I didn't use it and left it there. I'm going to fix it and commit again.

File: frontend/src/infrastructure/view/components/atoms/Image/types.ts (hunk 2 above)
Comment (Samuel-AQ) [reply frontend/src/infrastructure/view/components/atoms/Image/types.ts]: It's the same issue. I'm going to fix it and commit again.

**Review by javidiazdom — APPROVED** [review]

**Pull request merged**